tests/client/core/i18n/TestI18nProvider.js

```diff
@@ -22,7 +22,7 @@ class OuterComponent extends Component {
 
 class InnerComponent extends Component {
   static propTypes = {
-    i18n: PropTypes.object.isRequired,
+    i18n: PropTypes.shape({}).isRequired,
```

Comment:
oh, hah, I did remove them! And I did indeed forgot these, thanks. 
![owl](https://cloud.githubusercontent.com/assets/55398/18529762/15e12c5e-7a94-11e6-99dd-aa5b0592d0a0.jpg)